src/Symfony/Component/Console/Resources/completion.bash

```diff
@@ -6,8 +6,6 @@
 # https://symfony.com/doc/current/contributing/code/license.html
 
 _sf_{{ COMMAND_NAME }}() {
-    # Use newline as only separator to allow space in completion values
-    IFS=$'\n'
```

Comment:
Can you try if `local IFS=$'\n'` also fixes the issue?

Based on a quick local test, the behavior (being able to autocomplete values with spaces) still works when declaring it as a local variable.

Reply:
PR updated to your suggestion @wouterj

Reply:
Thanks for taking over sorry summer holidays here